x-pack/plugins/security_solution/public/management/components/endpoint_response_actions_list/components/action_log_expanded_tray.tsx

```diff
@@ -188,7 +189,7 @@ export const ActionsLogExpandedTray = memo<{
         },
         {
           title: OUTPUT_MESSAGES.expandSection.parameters,
-          description: parametersList ? parametersList : emptyValue,
+          description: parametersList ? parametersList.join(', ') : emptyValue,
```

Comment:
Thanks for this change.

Q. Have we tested the UI when parameter is long? like a long file path? does it display ok?

Reply:
@paul-tavares I did a small test. I think it looks alright. No?
<img width="1009" alt="Screenshot 2023-03-29 at 15 37 29" src="https://user-images.githubusercontent.com/1849116/228556059-62a09aa2-e637-42b3-8c30-5c7004220c23.png">

Reply:
This is ok for now, but I feel like this will not scale well. Could you just do one more test with just a long piece of text with just letters? (want to make sure it also breaks in the middle of words)

Reply:
If any changes, I'll add them to the new PR. elastic/kibana/pull/154181